lib/service/Service.ts

```diff
@@ -96,11 +96,16 @@ class Service {
     argChecks.VALID_SWAP_CLIENT(args);
     const { currency, swapClient, tokenAddress, decimalPlaces } = args;
 
+    let address = tokenAddress;
+    if (args.swapClient === SwapClientType.Raiden && !!address) {
```

Comment:
Very minor but the if condition logic here should work the same without the `!!`.